common/utils.js

```diff
@@ -519,12 +519,12 @@ async function getDomainSeparator(config, chain, options) {
     return expectedDomainSeparator;
```

Comment:
maybe use a more descriptive title with the right tag, e.g. `refactor: forward chains config instead of the whole config`

Reply:
https://www.conventionalcommits.org/en/v1.0.0/#summary